searx/engines/google_images.py

```diff
@@ -66,10 +66,10 @@ def response(resp):
     dom = html.fromstring(resp.text)
 
     # parse results
-    for result in dom.xpath('//div[@data-ved]'):
+    for result in dom.xpath('//div[contains(@class, "rg_meta")]/text()'):
```

Comment:
May as well zero in on the one interesting `div` in the returned data without first filtering on its container